beta-src/src/state/game/game-api-slice.ts

```diff
@@ -164,43 +164,34 @@ const resetOrder = (state) => {
 
 const startNewOrder = (
   state,
-  { payload: { unitID, onTerritory } }: NewOrderPayload,
+  {
+    payload: { unitID, onTerritory, orderID, toTerritory, type },
+  }: NewOrderPayload,
 ) => {
   const {
     data: { data: gameData },
   } = current(state);
   const { currentOrders } = gameData;
-  const orderForUnit = currentOrders.find((order) => {
-    return order.unitID === unitID;
-  });
+  let id = orderID;
+  if (!orderID) {
```

Comment:
shouldn't this check be `if(!id)` instead since you'r setting the variable above on L175

Reply:
Yea this code is kinda weird, I'll update it.